<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySetupDialog</class>
 <widget class="QDialog" name="DisplaySetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>509</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup Display</string>
  </property>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="devicePathEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Baud</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Bits</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_3_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bits</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="stopBitCombo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="parityCombo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dataBitCombo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>470</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Ca&amp;ncel</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_4_2_2_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Rows</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_4_2_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Base</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_4_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Trans</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>190</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="displayNameEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>290</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="baudCombo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>134</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="rowsSpinBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>280</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="baseSpinBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>280</y>
     <width>50</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
  </widget>
  <widget class="QComboBox" name="transCombo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>110</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Current</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Previous</string>
    </property>
   </item>
  </widget>
  <widget class="Q3Table" name="rowsTable">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>430</width>
     <height>140</height>
    </rect>
   </property>
   <property name="numRows" stdset="0">
    <number>0</number>
   </property>
   <property name="numCols" stdset="0">
    <number>3</number>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Null Display</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TD3000 serial port</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TD3200 serial port</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TD3000u USB port</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LT9900u USB port</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unused</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unused</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unused</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unused</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Video Display</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
